features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/admin-dashboard/site/themes/wso2/templates/loganalyzer/initializer.jag

```diff
@@ -0,0 +1,16 @@
+<%
+jagg.initializer("loganalyzer", {
```

Comment:
Hi Tharik,

This license header fixes by the commits in https://github.com/wso2/carbon-apimgt/pull/2906

Thanks,
SajithD